scripts/reset_iks_api_key.sh

```diff
@@ -22,13 +23,25 @@ if [[ -z "${RESOURCE_GROUP_ID}" ]]; then
     exit 1
 fi
 
+if [[ -z "${PRIVATE_ENV}" ]]; then
+    PRIVATE_ENV=false
+fi
+
 # Login to ibmcloud with cli
 attempts=1
-until ibmcloud login -q -r "${REGION}" -g "${RESOURCE_GROUP_ID}" || [ $attempts -ge 3 ]; do
-    attempts=$((attempts+1))
-    echo "Error logging in to IBM Cloud CLI..." >&2
-    sleep 5
-done
+if [ "$PRIVATE_ENV" ]; then
+    until ibmcloud login -q -r "us-south" -g "${RESOURCE_GROUP_ID}" -a private.cloud.ibm.com || [ $attempts -ge 3 ]; do
```

Comment:
looks like you hard coded `"us-south"` here, but you used `"${REGION}"` in else statement

Reply:
<img width="831" alt="Screenshot 2024-06-12 at 1 14 01 PM" src="https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/assets/122446118/079b4dc0-cb71-4017-9c9b-21a5c9bae101">

Reply:
might be worth putting that as a code comment